benchmarks/init_middleware.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+var http = require('http')
+var Benchmark = require('benchmark')
+var suite = new Benchmark.Suite()
+var IncomingMessage = http.IncomingMessage
+var ServerResponse = http.ServerResponse
```

Comment:
Sorry, but I had understood that the arguments of "expressInit" middleware were instances of IncomingMessage and ServerResponse. Was I wrong?